runtime/doc/index.txt

```diff
@@ -1144,6 +1144,7 @@ tag		command		action ~
 |:bNext|	:bN[ext]	go to previous buffer in the buffer list
 |:ball|		:ba[ll]		open a window for each buffer in the buffer list
 |:badd|		:bad[d]		add buffer to the buffer list
+|:balt|		:balt		like ":badd" and set the alternate file
```

Comment:
This is missing in Vim. Keep it anyway.

Reply:
Vim will soon have this as of vim/vim#7819. :+1: